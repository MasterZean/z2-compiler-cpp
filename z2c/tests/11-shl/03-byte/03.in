Byte{0b00000001u8} << 1
Byte{0b00000010u8} << 1
Byte{0b00000100u8} << 1
Byte{0b00001000u8} << 1
Byte{0b00010000u8} << 1
Byte{0b00100000u8} << 1
Byte{0b01000000u8} << 1
Byte{0b10000000u8} << 1

