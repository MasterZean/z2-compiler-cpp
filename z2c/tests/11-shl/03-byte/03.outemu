Byte{2}
Byte{4}
Byte{8}
Byte{16}
Byte{32}
Byte{64}
Byte{128}
Byte{0}
