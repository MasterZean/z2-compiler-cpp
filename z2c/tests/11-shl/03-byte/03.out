(uint8)(1u) << 1;
(uint8)(2u) << 1;
(uint8)(4u) << 1;
(uint8)(8u) << 1;
(uint8)(16u) << 1;
(uint8)(32u) << 1;
(uint8)(64u) << 1;
(uint8)(128u) << 1;
